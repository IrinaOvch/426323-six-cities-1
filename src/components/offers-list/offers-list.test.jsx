import React from 'react';
import renderer from 'react-test-renderer';
import OffersList from './offers-list.jsx';

const mock = [
  {
    id: 1,
    title: `Beautiful & luxurious studio at great location`,
    img: `img/apartment-01.jpg`,
    price: 120,
    isPremium: true,
    rating: 4,
    type: `Apartment`,
    isInBookmarks: false,
  }, {
    id: 1,
    title: `Beautiful & luxurious studio at great location`,
    img: `img/apartment-01.jpg`,
    price: 120,
    isPremium: true,
    rating: 4,
    type: `Apartment`,
    isInBookmarks: false,
  },
];

it(`Renders correctly`, () => {
  const offersList = renderer.create(
      <OffersList
        offers={mock}
      />
  ).toJSON;

  expect(offersList).toMatchSnapshot();
});
