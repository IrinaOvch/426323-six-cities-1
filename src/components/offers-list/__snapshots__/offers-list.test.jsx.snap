// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render OffersList correctly 1`] = `[Function]`;
